examples/weight-init/README.md

```diff
@@ -0,0 +1,93 @@
+# Weight Initialization
+
+Table of Contents
+=================
+* [Introduction](#introduction)
+* [Selecting a weight initialization strategy](#selecting-a-weight-initialization-strategy)
+* [Experiments with LeNet on MNIST](#experiments-with-lenet-on-mnist)
+
+## Introduction
+
+Weight initialization can critically affect the speed at which a neural network is able to learn. Under certain circumstances,
```

Comment:
Typically, you put each sentence on a different line in a Markdown file rather than chopping off the sentences at 80 characters. The former scheme makes it easier to review changes later, while the latter scheme makes it harder. Neither should affect the way the paragraph is actually displayed.